contributing/community/releases.rst

```diff
@@ -150,6 +150,10 @@ of Symfony to the next one.
 Whenever keeping backward compatibility is not possible, the feature, the
 enhancement or the bug fix will be scheduled for the next major version.
 
+However, features marked with the ``@experimental`` tag are **not** part of the
+backward compatibility promise and their APIs can change from one minor version
```

Comment:
@Wirone : if that's true the occurences of "backwards compatibility" in this PR should be removed.